p2p/conn/evil_secret_connection_test.go

```diff
@@ -222,12 +223,14 @@ func (c *evilConn) signChallenge() []byte {
 
 	b := &buffer{}
 	c.secretConn = &SecretConnection{
-		conn:       b,
-		recvBuffer: nil,
-		recvNonce:  new([aeadNonceSize]byte),
-		sendNonce:  new([aeadNonceSize]byte),
-		recvAead:   recvAead,
-		sendAead:   sendAead,
+		underlyingConn: b,
+		connReader:     b,
+		connWriter:     bufio.NewWriterSize(b, 65536),
```

Comment:
This is the new part, agree, the size of the buffer is something to have a as constant or at least discuss?